code/modules/wiremod/components/hud/click_intercept.dm

```diff
@@ -0,0 +1,64 @@
+/**
+ * # Click Intercept Component
+ *
+ * When activated intercepts next click and outputs clicked atom.
+ * Requires a BCI shell.
+ */
+
+/obj/item/circuit_component/click_intercept
+	display_name = "Click Intercept"
```

Comment:
"Click" is fairly OOC, see if you can come up with a better IC term.

Reply:
I thought of point instead of click but we also have "point" verb soooo